pkg/daemon/ceph/client/image.go

```diff
@@ -115,6 +115,24 @@ func DeleteImage(context *clusterd.Context, clusterName, name, poolName string)
 	return nil
 }
 
+func ResizeImage(context *clusterd.Context, clusterName, name, poolName, monitors, keyring string, size int64) error {
```

Comment:
Unit tests?

Reply:
Yes, I planed to add tests for resize today or next day.